lib/utils.js

```diff
@@ -251,9 +251,43 @@ function getPackageVersion (pkgName) {
   return pkgInfo.version;
 }
 
+/**
+ * Pulls the initial values of Appium settings from the given capabilities argument.
```

Comment:
I'd like a link to http://appium.io/docs/en/advanced-concepts/settings/index.html so we know what "Settings" are.

Reply:
see below